exec-unittests

```diff
@@ -127,8 +127,18 @@ fi
 _cov_report="${_cov_report_kind}:${_cov_report_name}"
 
 if [[ -z "${subtst}" || "${subtst}" == "python" ]]; then
-    _pytest_majors="pbench.test.unit.common"
+    _python_majors=""
+    _pytest_majors=""
     for _major in ${major_list}; do
+        if [[ ${_major} == "client" ]]; then
+            # No python tests for client major sub-system
+            continue
+        fi
```

Comment:
That's incorrect. `lib/pbench/test/unit/client` exists, with two test modules and a handful of test cases for create and connect:

```
(.venv39) dbutenho 07:40 audit:audit tox client
[...]
Running client python3-based unit tests via pytest

===================================================================== test session starts ======================================================================
platform linux -- Python 3.9.7, pytest-7.1.3, pluggy-1.0.0
cachedir: /var/tmp/dbutenho/tox/py39/.pytest_cache
rootdir: /home/dbutenho/github/audit, configfile: pytest.ini
plugins: requests-mock-1.10.0, mock-3.10.0, helpers-namespace-2019.1.8, freezegun-0.4.2, forked-1.4.0, cov-4.0.0, xdist-2.5.0
gw0 [24] / gw1 [24] / gw2 [24] / gw3 [24]
........................
```

Reply:
Ugh, I totally messed that up.  Fixing...

Reply:
Restored.

But ... this is weird.

The client sub-system is really part of server.  The majors are the top-level directories, agent, server, and dashboard.

What are the "clients"?

Reply:
I didn't spend a lot of time thinking about packaging at the time; I wanted to get a start on the basic framework for functional testing.

It's associated with the server, but the idea is that we want this code *outside* the server, not *inside* the server. It's a pseudo-separate package, and it has its own unit tests, but currently we're only building it as part of the server package. Yeah, that's really wrong, and it probably needs its own `setup.py` and packaging. Maybe we can think about that when we move to poetry.